src/components/application_manager/src/commands/mobile/system_request.cc

```diff
@@ -516,6 +515,11 @@ void SystemRequest::Run() {
   std::string file_name;
   if ((*message_)[strings::msg_params].keyExists(strings::file_name)) {
     file_name = (*message_)[strings::msg_params][strings::file_name].asString();
+    if (std::string::npos != file_name.find("/")) {
```

Comment:
@LuxoftAKutsan, yes, according to [Issue-975]https://github.com/smartdevicelink/sdl_core/issues/975) we should send INVALID_DATA only on SystemRequest that uploads a file containing '/'